src/pages/home.tsx

```diff
@@ -0,0 +1,113 @@
+                return
+            }
+            setFollowing(res.data);
+            setFPage(fPage+nextPage)
+        });
+    }
+
+    useEffect(() => {
+        searchRepos()
+        searchFollowing();
+    }, []);
+
+    return (
+        <section className={styles.main}>
+            <div className={styles.opts}>
+                <span onClick={e => switchDivs('repositories', 'following')}>Following</span>
+                <span onClick={e => switchDivs('following', 'repositories')}>Repositories</span>
+            </div>
+
+            <div className={styles.content}>
+                <div className={styles.following} id='following'>
+                    <ul>
+                        {
+                            following.map(item => {
+                                return (
+                                    <li key={Math.random()}>
+                                        <img src={item.avatar_url} />
+                                        <span>{item.login}</span>
+                                        <span> <a href={item.html_url} target='_blank'> View in github </a> </span>
+                                    </li>
+                                )
+                            })
+                        }
+                    </ul>
+                    <div className={styles.pageController}>
+                        <span onClick={e => {
+                            if(fPage <= 1) {
+                                return
+                            }
+                            searchFollowing(-1)
+                        }}> &#9664; </span>
+                        <span> {fPage} </span>
+                        <span onClick={e => searchFollowing(1)}> &#9654; </span>
+                    </div>
+                </div>
+
+                <div className={styles.repositories} id='repositories'>
+                    <ul>
+                        {
+                            repos.map(item => {
+                                return (
+                                    <li key={Math.random()}>
+                                        <span>{item.name}</span>
+                                        <span> <a href={item.html_url} target='_blank'> View in github </a> </span>
+                                        {item.homepage ? <span> <a href={item.homepage} target='_blank'>Homepage</a> </span> : null}
+                                    </li>
+                                )
+                            })
+                        }
+                    </ul>
+                    <div className={styles.pageController}>
+                        <span onClick={e => {
+                            if(fPage <= 1) {
+                                return
+                            }
+                            searchFollowing(-1)
+                        }}> &#9664; </span>
+                        <span> {fPage} </span>
+                        <span onClick={e => searchFollowing(1)}> &#9654; </span>
```

Comment:
```suggestion
                            if(rPage<= 1) {
                                return
                            }
                            searchRepos(-1)
                        }}> &#9664; </span>
                        <span> {rPage} </span>
                        <span onClick={e => searchRepos(1)}> &#9654; </span>
```

Estava alterando a pagina do "Following"